indico/modules/events/registration/templates/display/registration_summary.html

```diff
@@ -12,7 +12,7 @@
                    {% trans %}Your registration has been completed{% endtrans %}
                </div>
                {% trans -%}
-                   Everything is done.
+                   The process is over.
```

Comment:
Now that I see, we did get some complaints about this "Everything is done" message.
Maybe we can just omit it or replace it with something like "Thank you".